content/_index.md

```diff
@@ -11,15 +11,16 @@ title: OpenClarity
 
 <h3 class="hero-subtitle">OpenClarity delivers:</h3>
 <ul>
-<li>Agent-less detection and managements of vulnerabilities, exploits, malware and misconfigurations for VMs and Container Images</li>
+<li>Agent-less detection and management of vulnerabilities, exploits, malware, and misconfigurations for VMs and Container Images</li>
 <li>Capabilities for runtime scans of Kubernetes and CI/CD pipelines</li>
 <li>Comprehensive API security for internal and third-party APIs</li>
 </ul>
 
 <div class="mx-auto">
   {{< cta-button text="Learn more" url="/docs/" classes="bg-sky-blue text-white" >}}
 
-  {{< cta-button text="Install" url="/docs/getting-started/_index.md" classes="bg-black text-white" icon="fa-brands fa-github ml-2" >}}
+  {{< cta-button text="Try it now" url="/docs/getting-started/" classes="bg-sky-blue text-white" >}}
```

Comment:
I would keep the original color to visibly show the difference between "Learn more" and "Try it now"
```suggestion
  {{< cta-button text="Try it now" url="/docs/getting-started/" classes="bg-black text-white" >}}
```